tests/result/sflow.pcap.out

```diff
@@ -19,6 +19,6 @@ Patricia risk mask:   2/0 (search/found)
 Patricia risk:        0/0 (search/found)
 Patricia protocols:   6/0 (search/found)
 
-sFlow	9	1702	1
+Softether	9	1702	1
 
-	1	UDP 172.21.35.17:1027 -> 172.21.35.199:6343 [proto: 129/sFlow][ClearText][Confidence: DPI][cat: Network/14][9 pkts/1702 bytes -> 0 pkts/0 bytes][Goodput ratio: 78/0][109.01 sec][bytes ratio: 1.000 (Upload)][IAT c2s/s2c min/avg/max/stddev: 6002/0 13626/0 19002/0 4554/0][Pkt Len c2s/s2c min/avg/max/stddev: 186/0 189/0 214/0 9/0][Risk: ** Unidirectional Traffic **][Risk Score: 10][Risk Info: No client to server traffic][PLAIN TEXT (abcdefghijklmnopq)][Plen Bins: 0,0,0,0,88,11,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```

Comment:
Isn't that a regression? This flows seems a valid sFlow flow (to Wireshark, at least)

Reply:
True, didn't notice.